recipe/meta.yaml

```diff
@@ -27,33 +32,50 @@ build:
     - "$RPATH/libtorch_cpu.dylib"  # [osx]
     - "$RPATH/libc10.so"           # [linux]
     - "$RPATH/libc10.dylib"        # [osx]
+    - "$RPATH/c10.dll"             # [win]
+    - "$RPATH/torch_cpu.dll"       # [win] 
+
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    # libpng and libjpeg must be available at compilation time in order to be available,
+    # see https://github.com/pytorch/vision/tree/v0.11.3#image-backend
+    - jpeg      # [not win]
+    - libpng
+    - m2-patch  # [win]
+    - patch     # [not win]
+    - sysroot_linux-64 ==2.17         # [linux64]
   host:
     # GPU requirements
     - cudatoolkit {{ cudatoolkit }}*  # [pytorch_variant == "gpu"]
     # other requirements
     - python
+    - jpeg      # [not win]
+    - libpng
     - pip
+    - pillow >=5.3.0,!=8.3.*
+    - pytorch ==1.10.2
     - setuptools
-    - jpeg
-    - libpng
-    - pytorch 1.7.1  # [linux and not (s390x or aarch64)]
-    - pytorch 1.8.1  # [linux and (s390x or aarch64)]
+    # - ffmpeg
   run:
-    - _pytorch_select ==0.1             # [pytorch_variant == "cpu"]
-    - _pytorch_select ==0.2             # [pytorch_variant == "gpu"]
+    # - _pytorch_select * cpu*             # [pytorch_variant == "cpu"]
+    # - _pytorch_select * gpu*             # [pytorch_variant == "gpu"]
     # GPU requirements
     - {{ pin_compatible('cudatoolkit', max_pin='x.x') }}  # [pytorch_variant == "gpu"]
     # other requirements
     - python
-    - pillow >=4.1.1
-    - numpy >=1.11
-    - pytorch 1.7.1  # [linux and not (s390x or aarch64)]
-    - pytorch 1.8.1  # [linux and (s390x or aarch64)]
-    - six
+    - {{ pin_compatible('numpy') }}
+    - pillow >=5.3.0,!=8.3.*
+    - pytorch ==1.10.2
+    - setuptools
+    # - ffmpeg
```

Comment:
May want to add a comment with these links to explain why we're not requiring `ffmpeg`: https://github.com/pytorch/vision/issues/5419#issuecomment-1193483864
https://github.com/conda-forge/torchvision-feedstock/commit/4ec4b5f4eb4889dbb1f8da34662ea622bb4b3828

Reply:
I've added comments